stacker_blueprints/vpc.py

```diff
@@ -352,7 +352,6 @@ def create_nat_instance(self, zone_id, subnet_name):
             t.add_resource(
                 ec2.Instance(
                     instance_name,
-                    Condition="UseNatInstances",
```

Comment:
If this changes the resulting CFT json, we cannot safely merge this.

Reply:
It only changes the underlying cloudformation if someone were using it with NatInstances already (see the if statement above).  Unfortunately if they WERE using that, it wouldn't work, because of this bug.  I think hardly anyone is using NatInstances any longer. @russellballestrini

Reply:
I see, awesome. I was thinking this was the stacks repo, too.